src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 
 const ActionButtons = ({ actionButtons }) => {
   if (actionButtons) {
     return (
       <React.Fragment>
-        <a className="dummy-link">Edit</a> <a className="dummy-link">Delete</a>
+        <a className="dummy-link">Edit</a> <Link to="https://www.google.com/">Delete</Link>
```

Comment:
I must have added that google link there by mistake. This is because this file is not even important to the changes that I was making about displaying live data on the Concepts HEAD version card. I have removed that link, reverted or the changes in that file and updated this PR. Thanks for catching that.